test/tableSTAT.js

```diff
@@ -0,0 +1,707 @@
+  0x00, 0x00,                       // 000E [1] uint16  axisOrdering
+                              // 0024         Offset  axisValueOffsets[6]
+  0x00, 0x0c,                       // 0000   axisValueOffsets[0]
+  0x00, 0x18,                       // 0002   axisValueOffsets[1]
+  0x00, 0x24,                       // 0004   axisValueOffsets[2]
+  0x00, 0x30,                       // 0006   axisValueOffsets[3]
+  0x00, 0x3c,                       // 0008   axisValueOffsets[4]
+  0x00, 0x48,                       // 000A   axisValueOffsets[5]
+                              // 0030  000C   axisValue[0]
+  0x00, 0x01,                       // 0000     uint16  format
+  0x00, 0x00,                       // 0002     uint16  axisIndex
+  0x00, 0x02,                       // 0004     uint16  flags
+  0x01, 0x03,                       // 0006     uint16  valueNameID
+  0x00, 0x00, 0x00, 0x00,           // 0008     Fixed(16.16)  value  0.0
+                              // 003C  0018   axisValue[1]
+  0x00, 0x01,                       // 0000     uint16  format
+  0x00, 0x01,                       // 0002     uint16  axisIndex
+  0x00, 0x00,                       // 0004     uint16  flags
+  0x01, 0x01,                       // 0006     uint16  valueNameID
+  0x01, 0x2c, 0x00, 0x00,           // 0008     Fixed(16.16)  value  300.0
+                              // 0048  0024   axisValue[2]
+  0x00, 0x01,                       // 0000     uint16  format
+  0x00, 0x01,                       // 0002     uint16  axisIndex
+  0x00, 0x00,                       // 0004     uint16  flags
+  0x01, 0x02,                       // 0006     uint16  valueNameID
+  0x01, 0x5e, 0x00, 0x00,           // 0008     Fixed(16.16)  value  350.0
+                              // 0054  0030   axisValue[3]
+  0x00, 0x01,                       // 0000     uint16  format
+  0x00, 0x01,                       // 0002     uint16  axisIndex
+  0x00, 0x00,                       // 0004     uint16  flags
+  0x01, 0x03,                       // 0006     uint16  valueNameID
+  0x01, 0x90, 0x00, 0x00,           // 0008     Fixed(16.16)  value  400.0
+                              // 0060  003C   axisValue[4]
+  0x00, 0x01,                       // 0000     uint16  format
+  0x00, 0x01,                       // 0002     uint16  axisIndex
+  0x00, 0x00,                       // 0004     uint16  flags
+  0x01, 0x04,                       // 0006     uint16  valueNameID
+  0x02, 0x58, 0x00, 0x00,           // 0008     Fixed(16.16)  value  600.0
+                              // 006C  0048   axisValue[5]
+  0x00, 0x01,                       // 0000     uint16  format
+  0x00, 0x01,                       // 0002     uint16  axisIndex
+  0x00, 0x00,                       // 0004     uint16  flags
+  0x01, 0x05,                       // 0006     uint16  valueNameID
+  0x02, 0xbc, 0x00, 0x00            // 0008     Fixed(16.16)  value  700.0
+];                                  // 0078
```

Comment:
Instead of including the binary data in the source code, can we just include the font file in the repo and read the table directly in the tests?